src/AccordionItemHeading/AccordionItemHeading.tsx

```diff
@@ -23,9 +35,26 @@ export default class AccordionItemHeading extends React.Component<
     };
 
     handleKeyPress = (evt: React.KeyboardEvent<HTMLDivElement>): void => {
-        if (evt.charCode === 13 || evt.charCode === 32) {
-            evt.preventDefault();
-            this.handleClick();
+        const keyCode = evt.which.toString();
+        const parentSelector = '[data-accordion]';
+        const parentAccordion = getClosest(evt.target, parentSelector);
+
+        switch (keyCode) {
+            case '13':
+            case '32':
+                evt.preventDefault();
+                this.handleClick();
+                break;
+            case '36':
+                // home
+                focusNextHeading(evt, parentAccordion.firstChild);
```

Comment:
I don't think `firstChild` will necessarily work, because that markup is not strictly enforced. It will have to be `parentAccordion.querySelectorAll(itemSelector)[0]` or similar.